vm/chef/cookbooks/percona/files/percona-setup

```diff
@@ -63,7 +63,8 @@ readonly mysql_root_password="$(get_attribute_value "MYSQL_ROOT_PASSWORD")"
 readonly percona_disk_name="$hostname-data"
 
 # Add Percona Toolkit, if user requests it.
-if [[ "${install_percona_toolkit}" == "true" ]]; then
+install_percona_toolkit="$(get_attribute_value "INSTALL_PERCONA_TOOLKIT")"
```

Comment:
This is already defined in the 61 line, right?

https://github.com/GoogleCloudPlatform/click-to-deploy/blob/eedb6832e5540baad2043daa238c53324b00a5f9/vm/chef/cookbooks/percona/files/percona-setup#L61